third_party/farmhash/farmhash_support_cuda.patch

```diff
@@ -0,0 +1,276 @@
+diff --git a/src/farmhash.cc b/src/farmhash_gpu.h
```

Comment:
Can this be formatted as a git patch, with a commit message?  That way the patch is self describing.

Reply:
Done.